src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/memory_LSTMCell.hpp

```diff
@@ -5,28 +5,29 @@
 #pragma once
 
 #include "common_test_utils/test_common.hpp"
-#include "shared_test_classes/base/layer_test_utils.hpp"
-#include <ie_core.hpp>
-
-namespace SubgraphTestsDefinitions {
-typedef std::tuple<
-    ngraph::helpers::MemoryTransformation,   // Apply Memory transformation
-    std::string,                        // Target device name
-    InferenceEngine::Precision,         // Network precision
-    size_t,                             // Input size
-    size_t,                             // Hidden size
-    std::map<std::string, std::string>  // Configuration
-> memoryLSTMCellParams;
-
-class MemoryLSTMCellTest : virtual public LayerTestsUtils::LayerTestsCommon,
-    public testing::WithParamInterface<memoryLSTMCellParams> {
+#include "ov_models/utils/ov_helpers.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<ngraph::helpers::MemoryTransformation,  // Apply Memory transformation
+                   std::string,                            // Target device name
+                   ov::element::Type,                      // Input element type
+                   size_t,                                 // Input size
+                   size_t,                                 // Hidden size
+                   ov::AnyMap                              // Configuration
+                   >
+    memoryLSTMCellParams;
+
+class MemoryLSTMCellTest : virtual public ov::test::SubgraphBaseStaticTest,
+                           public testing::WithParamInterface<memoryLSTMCellParams> {
 private:
     // you have to Unroll TI manually and remove memory untill ngraph supports it
```

Comment:
please, fix comment (`untill ngraph`)

Reply:
It seems TI(TensorIterator) has been unrolled in `switch_to_friendly_model()`, please see blow dumped graph:

Setup models:
![image](https://github.com/openvinotoolkit/openvino/assets/31196718/f7d88001-70bd-47ce-a74f-97de0700cafd)

`switch_to_friendly_model()` model:
![image](https://github.com/openvinotoolkit/openvino/assets/31196718/31adc1a7-06e3-4a84-af35-74e80ce47eb8)

Do I understand correctly? @vurusovs

Reply:
Got it!